.github/workflows/build.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: build
+on: [ push, pull_request ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Not keen on this. Could we use bionic please or focal if it exists

Reply:
FYI GitHub is only using LTS releases here. 'Latest' was recently upgraded to switch from 18.04 to 20.04. 

I'll switch it to 20.04 LTS so it's explicit.

Reply:
Added in [`b0fb685`](https://github.com/cucumber/cucumber-rails/compare/51ac21f81d8fdcb88d49c699e38c19d9f0a2f30b..b0fb68513929b07ca88cd624da6b4007017dc13e)